code/src/pages/NotFound.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+const NotFound = () => {
```

Comment:
Your component doesn't work when we pass into url `movieId` that doesn't exist. In order to achieve that, you need to create conditional check inside `MovieDetails` component and check if `movieDetails` state property is correct.
Apart from that case, great job :)